src/Datadog.Instrumentation/Datadog.Instrumentation.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version>1.28.2</Version>
+    <Title>Datadog APM Auto-instrumentation Assets</Title>
+    <Description>Auto-instrumentation assets for Datadog APM</Description>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <NoWarn>NU5100;NU5128</NoWarn>
+    <PackageOutputPath>bin\$(Configuration)\packages</PackageOutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="home\**\*.*" Pack="true" PackagePath="content\datadog;contentFiles\any\any\datadog">
+      <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+      <PackageCopyToOutput>true</PackageCopyToOutput>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Content Update="home\**\readme.txt" Pack="false">
+      <CopyToOutputDirectory>Never</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Datadog.Trace\Datadog.Trace.csproj" />
```

Comment:
Does this end up as a nuget package reference in the output? Or will the nuget include `Datadog.Trace.dll`?

Reply:
Looks like a nuget reference to `Datadog.Trace >= 1.28.2`. Neat!

Reply:
Yep, NuGet package reference in the output, and this NuGet package doesn't provide any ref/lib assemblies itself 👍🏼